### ISSUE TITLE
History -> Summary in QT does not work with FX rate disabled

### ISSUE THREAD

**Issue by gits7r** [spesmilo/electrum#7405]
In previous versions it was possible to go to History -> Summary and see the total BTC in, total BTC out with N/A to FX rate.
Now this menu button will not work at all if there is no FX rate. I was using it to count the total turnover in BTC (regardless fiat rate) and number of transactions. Is there any reason why it doesn't work any more without an FX rate?

**Comment by SomberNight** [comment]
The dialog is mainly for showing fiat data.

The original motivation for adding it was to help us with tax accounting. In large part it is there to help with the headache of capital gains tax...
IIRC it was changed recently because there was some issue with the cap gains calculation; and also we needed some more fields.

![Capture](https://user-images.githubusercontent.com/29142493/125812783-b16dbe58-ff28-4c30-ab5d-8f3d09b9b58f.PNG)

Anyway, yes, I suppose the dialog could still work without FX rates with most fields disabled.

**Comment by gits7r** [comment]
Yes, and we can just replace the FIAT fields with N/A if FX rate is disabled, just like we did before. Now, one can know the total turnover (in BTC) of a wallet. And N/A for realized capital gains of course and unrealized capital gains / acquisition price.
